src/HttpClient/ForbiddingHttpClient.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace eLife\ApiClient\HttpClient;
+
+use eLife\ApiClient\HttpClient;
+use GuzzleHttp\Promise\PromiseInterface;
+use LogicException;
+use Psr\Http\Message\RequestInterface;
+use function GuzzleHttp\Psr7\str;
+
+final class ForbiddingHttpClient implements HttpClient
+{
+    public function send(RequestInterface $request) : PromiseInterface
+    {
+        throw new LogicException('Forbidden call: '.str($request));
```

Comment:
Ah, I was thinking of this an an 'offline' client, rather than making sure that no requests are made. But I'm happy to go for that (fits this name) and have a new exception (extending `HttpProblem`).